integration/class-enable-mastodon-apps.php

```diff
@@ -300,67 +366,53 @@ public static function api_statuses_external( $statuses, $args ) {
 
 		$activitypub_statuses = array_map(
 			function ( $item ) use ( $account ) {
-				$object = $item['object'];
-				if ( ! isset( $object['type'] ) || 'Note' !== $object['type'] ) {
-					return null;
-				}
+				return self::activity_to_status( $item, $account );
+			},
+			$posts['orderedItems']
+		);
 
-				$status = new Status();
-				$status->id         = Mastodon_API::remap_url( $object['id'] );
-				$status->created_at = new DateTime( $object['published'] );
-				$status->content    = $object['content'];
-				$status->account    = $account;
+		return $activitypub_statuses;
+	}
 
-				if ( ! empty( $object['inReplyTo'] ) ) {
-					$status->in_reply_to_id = $object['inReplyTo'];
-				}
+	public static function api_get_replies( $context, $post_id, $url ) {
+		$response = Http::get( $url, true );
```

Comment:
should we us https://github.com/Automattic/wordpress-activitypub/blob/74403ff37f30f1cdd28a48d7b7993900bf370718/includes/class-http.php#L166 instead?

Reply:
Nice, ok!